static/docs/commands-reference/status.md

```diff
@@ -53,19 +53,40 @@ This says that no differences were detected, and therefore that no stages would
 be run again if `dvc repro` were executed.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in both _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally, a status word is shown describing the
+changes in checksum or status for both local and cloud workspace. Given list
+provides a reference to both, the status of the DVC-file and differences in the
```

Comment:
`the status of the DVC-file as well as dependencies and outputs:`

Reply:
What is a "cloud workspace"?
These last 2 sentences in the paragraph are confusing. They seem unnecessarily long and maybe repetitive. Feel free to break up the paragraph in 2 if needed or even re-write the whole thing from scratch, as long as the explanation is clear.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/416/files#r303261384